.github/workflows/test-suite.yml

```diff
@@ -6,7 +6,7 @@ on:
     inputs:
       terraform_matrix:
         description: 'Terraform version matrix (JSON array)'
-        default: '["1.8.x", "1.7.x", "1.6.x", "1.5.x", "1.4.x", "1.3.x", "1.2.x"]'
+        default: '["1.9.x", "1.8.x", "1.7.x", "1.6.x"]'
```

Comment:
We're still supporting versions as low as 1.3.x as per https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2370/files until 9/30, could you confirm why these are being removed?

Reply:
my bad, reverted, thx